pkg/console/subresource/deployment/deployment.go

```diff
@@ -66,11 +66,11 @@ var volumeConfigList = []volumeConfig{
 	},
 }
 
-func DefaultDeployment(cr *v1alpha1.Console, cm *corev1.ConfigMap, serviceCAConfigMap *corev1.ConfigMap, sec *corev1.Secret) *appsv1.Deployment {
+func DefaultDeployment(cr *v1alpha1.ConsoleOperatorConfig, cm *corev1.ConfigMap, serviceCAConfigMap *corev1.ConfigMap, sec *corev1.Secret) *appsv1.Deployment {
 	labels := util.LabelsForConsole()
 	meta := util.SharedMeta()
 	meta.Labels = labels
-	replicas := cr.Spec.Count
+	replicas := int32(2)
```

Comment:
Not sure if hardcoding this is better thought :-/
If so lets make it a `const` ?